### DISCUSSION TITLE
Pro Tip: You can underclock madmax without a performance hit

### DISCUSSION THREAD

**Discussion by trhr** [madMAx43v3r/chia-plotter#878] [Show and tell]
We have a fleet (17 units) of r730s that chug 600W of pixie juice on Performance settings. We set a hard cap of 275W for each unit and didn't see much, if any, impact on performance.

per day stats.
![image](https://user-images.githubusercontent.com/5786272/127756766-d576b995-0863-42b4-a0e6-8ca01144cb9f.png)

**Comment by idog8818** [comment]
Mine running at ~350W peak, if I limited to eg 250W I can see plotting time is increasing hence not good for me.

**Reply by trhr** [reply]
guess it varies. we have 2133mhz ram and 56 cores per box, so I think we're bumping into our RAM speed limits. our CPUs peaked at 60% at full power.

**Reply by idog8818** [reply]
I'm pretty similar. 28C56T. For some reason 2133mhz running at 1800. CPU will be nearly 100%
